src/core/CMakeLists.txt

```diff
@@ -2557,14 +2531,15 @@ if (APPLE)
 endif()
 
 target_link_libraries(qgis_core
-  ${QT_VERSION_BASE}::Core
-  ${QT_VERSION_BASE}::Gui
-  ${QT_VERSION_BASE}::Xml
-  ${QT_VERSION_BASE}::Widgets
-  ${QT_VERSION_BASE}::Svg
-  ${QT_VERSION_BASE}::Network
-  ${QT_VERSION_BASE}::Sql
-  ${QT_VERSION_BASE}::Concurrent
+  Qt::Core
+  Qt::Gui
+  Qt::Xml
+  Qt::Widgets
+  Qt::Svg
+  Qt::Network
+  Qt::Sql
+  Qt::Concurrent
+  Qt::Positioning
```

Comment:
Note to self: this could cause troubles when generating an exported target https://doc.qt.io/qt-6/cmake-qt5-and-qt6-compatibility.html#pitfalls-when-using-versionless-targets (which is still on my bucket list)
Better undo ...